src/algorithm/ATOMIC.cpp

```diff
@@ -35,6 +35,10 @@ ATOMIC::ATOMIC(const RunParams& params)
 
   setComplexity(Complexity::N);
 
+  setNestedLoops(0);
+  setArrayDimensions(1);
+  setNumArrays(1);
```

Comment:
In most cases this is simple, but in some cases as mentioned for LTIMES there are arrays of differing dimensionalities. Do you have a good idea of what you want to count here or not?